cmd/mailchain/internal/settings/senders.go

```diff
@@ -5,19 +5,23 @@ import (
 	"github.com/mailchain/mailchain/cmd/internal/settings/values"
 	"github.com/mailchain/mailchain/internal/protocols"
 	"github.com/mailchain/mailchain/internal/protocols/ethereum"
+	"github.com/mailchain/mailchain/internal/protocols/substrate"
 	"github.com/mailchain/mailchain/sender"
 	"github.com/pkg/errors"
 )
 
 func senders(s values.Store) *Senders {
 	return &Senders{
 		clients: map[string]SenderClient{
-			"ethereum-rpc2-" + ethereum.Goerli:  ethereumRPC2Sender(s, ethereum.Goerli),
-			"ethereum-rpc2-" + ethereum.Kovan:   ethereumRPC2Sender(s, ethereum.Kovan),
-			"ethereum-rpc2-" + ethereum.Mainnet: ethereumRPC2Sender(s, ethereum.Mainnet),
-			"ethereum-rpc2-" + ethereum.Rinkeby: ethereumRPC2Sender(s, ethereum.Rinkeby),
-			"ethereum-rpc2-" + ethereum.Ropsten: ethereumRPC2Sender(s, ethereum.Ropsten),
-			protocols.Ethereum + "-relay":       relaySender(s, protocols.Ethereum),
+			"ethereum-rpc2-" + ethereum.Goerli:           ethereumRPC2Sender(s, ethereum.Goerli),
+			"ethereum-rpc2-" + ethereum.Kovan:            ethereumRPC2Sender(s, ethereum.Kovan),
+			"ethereum-rpc2-" + ethereum.Mainnet:          ethereumRPC2Sender(s, ethereum.Mainnet),
+			"ethereum-rpc2-" + ethereum.Rinkeby:          ethereumRPC2Sender(s, ethereum.Rinkeby),
+			"ethereum-rpc2-" + ethereum.Ropsten:          ethereumRPC2Sender(s, ethereum.Ropsten),
+			protocols.Ethereum + "-relay":                relaySender(s, protocols.Ethereum),
+			"substrate-rpc-" + substrate.EdgewareTestnet: substrateRPCSender(s, substrate.EdgewareTestnet),
```

Comment:
It looks like `EdgewareTestnet` should now be called `EdgewareBerlin` see https://polkadot.js.org/apps/#/explorer with an address of `wss://berlin1.edgewa.re`

In addition there can now be `EdgewareMainnet` with an endpoint of `wss://mainnet1.edgewa.re`